extensions/analyticsdx-vscode-templates/src/autoInstall/completions.ts

```diff
@@ -26,11 +27,9 @@ export class AutoInstallVariableCompletionItemProviderDelegate extends VariableR
 
   public isSupportedLocation(location: Location) {
     return (
-      location.isAtPropertyKey &&
-      location.matches(['configuration', 'appConfiguration', 'values', '*']) &&
-      // this makes sure the completion only show in prop names directly under "values" (and not in prop names in object
+      // makes sure the completion only show in prop names directly under "values" (and not in prop names in object
       // values under "values")
-      location.path.length === 4
+      location.isAtPropertyKey && locationMatches(location, ['configuration', 'appConfiguration', 'values', '*'])
```

Comment:
See the change in jsoncUtils.ts down below for what all these `locationMatches()` changes are about.